src/Session.h

```diff
@@ -222,7 +221,7 @@ class Session {
     bool SendSpectralProfileData(int file_id, int region_id, bool stokes_changed = false);
 
     FileSettings _file_settings;
-    std::unordered_map<int, tbb::concurrent_queue<std::pair<CARTA::SetImageChannels, uint32_t>>> _set_channel_queues;
+    std::unordered_map<int, concurrent_queue<std::pair<CARTA::SetImageChannels, uint32_t>>> _set_channel_queues;
```

Comment:
shouldn't this be `carta::concurrent_queue`? Since `Session` is not part of the `carta` namespace yet